source/extensions/filters/http/grpc_field_extraction/message_converter/message_converter_utility.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <string>
+
+#include "envoy/buffer/buffer.h"
+
+#include "source/common/buffer/buffer_impl.h"
+#include "source/common/buffer/zero_copy_input_stream_impl.h"
+
+#include "absl/status/statusor.h"
+#include "proto_field_extraction/message_data/message_data.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace GrpcFieldExtraction {
+
+using CreateMessageDataFunc =
+    std::function<std::unique_ptr<Protobuf::field_extraction::MessageData>()>;
+
+// The output for the parseGrpcMessage function.
+struct ParseGrpcMessageOutput {
```

Comment:
nit: How 'bout just "ParsedGrpcMessage"?

Reply:
We already have a method `parseGrpcMessage` and The `Output` suffix helps differeniate them and show this struct is its output.

Reply:
Yeah, I'm suggesting "ParsedGrpcMessage" not "ParseGrpcMessage" In other words, there's trailing "d" in the first word. In general it's better to use named which describe what a class is (in this case, it's the parsed version of a grpc message) than to name if after the method which returns it.

Reply:
DONE.